agents-manage-ui/src/components/projects/new-project-dialog.tsx

```diff
@@ -42,16 +38,7 @@ export function NewProjectDialog({
 
   return (
     <Dialog open={open} onOpenChange={setOpen}>
-      {(children || triggerButton) && (
-        <DialogTrigger asChild>
-          {children || triggerButton || (
-            <Button>
-              <Plus className="mr-2 h-4 w-4" />
-              Create Project
-            </Button>
-          )}
-        </DialogTrigger>
-      )}
+      {children && <DialogTrigger asChild>{children}</DialogTrigger>}
```

Comment:
to make works this code
<img width="589" height="512" alt="image" src="https://github.com/user-attachments/assets/54620eac-9d7e-4593-ba91-7fce5a1e1f67" />